### PR TITLE
RDS Aurora, Meta/Docs gen tool

### PR DESCRIPTION
**Pull request by ndrest-amzn**
## Overview

RDS Aurora MySQL
RDS Aurora Postgress
RDS MS SQL 
CFN templates + readme

First iteration of templates for these services.

## Testing

To be vetted, and tested by author.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ndrest-amzn** [comment]
Update docs/README.md - Added  links to two tools for helping auto generate the service broker metadata found in the template, and and generate the Readme.md required for a template

File: templates/aurorapostgresql/template.yaml [hunk 1]
```diff
@@ -0,0 +1,1225 @@
...
+###############################################################################   
+Metadata:
+  AWS::ServiceBroker::Specification:
+    Version: 1.0
+    Tags:
+    - AWS    
+    - Aurora
+    - PostgreSQL
+    - database
+    Name: rdspostgresql
```
Comment (jaymccon) [inline templates/aurorapostgresql/template.yaml]: this must be changed to match the folder name `aurorapostgresql`

File: templates/auroramysql/template.yaml [hunk 2]
```diff
@@ -0,0 +1,1400 @@
...
+
+  VpcId:
+    Description: The ID of the VPC to launch the RDS instance into
+    Type: AWS::EC2::VPC::Id
+
+  ServerlessMinCapacityUnit:
+    Description: The minimum capacity for an Aurora DB cluster in serverless DB engine mode. The minimum capacity must be less than or equal to the maximum capacity.
+    Type: Number
+    Default: 2
+    AllowedValues:
```
Comment (jaymccon) [inline templates/auroramysql/template.yaml]: The openshift ui doesn't like AllowedValues combined with numbers. Either change the type to `String` or remove `AllowedValues`

File: templates/aurorapostgresql/template.yaml (hunk 1 above)
Comment (ndrest-amzn) [reply templates/aurorapostgresql/template.yaml]: changed it to match the folder name aurorapostgresql

File: templates/auroramysql/template.yaml (hunk 2 above)
Comment (ndrest-amzn) [reply templates/auroramysql/template.yaml]: Changed AllowedValues from number to string.  Tested in AWS console and ok.

**Comment by jaymccon** [comment]
Great stuff! Thanks a lot for this contribution @ndrest-amzn

**Pull request merged**